torch/distributions/beta.py

```diff
@@ -48,6 +49,12 @@ def expand(self, batch_shape, _instance=None):
     def mean(self):
         return self.concentration1 / (self.concentration1 + self.concentration0)
 
+    @property
+    def mode(self):
+        mode = (self.concentration1 - 1) / (self.concentration0 + self.concentration1 - 2)
+        mode[(self.concentration0 <= 1) & (self.concentration1 <= 1)] = nan
```

Comment:
Wouldn't one of the endpoints be the mode in this case?  Or perhaps `torch.finfo(self.concentration0.dtype).tiny`?  I don't have a clear understanding of open/closed sets or boundaries in a distribution library; I suspect a proper treatment of boundaries would require code complexity that would dwarf all of the existing torch.distributions, and therefore I've punted and avoided any boundary logic.

Reply:
Yes, good point. Fixed in dd871705d31dd1052ab258e07aaa2ded0dbc0734.